src/ZXing.Net.Maui/ZXing.Net.MAUI.Controls/Controls/BarcodeGeneratorView.cs

```diff
@@ -7,7 +7,6 @@
 using System.Text;
 using System.Threading.Tasks;
 using ZXing.Net.Maui;
-using Microsoft.Maui.Graphics;
```

Comment:
Should we fix this in the commit that added ZXing in?

Reply:
No. Why?

Reply:
What do you mean "non-related commit"?
I have 2 commits in this branch: 
first adds ZXing as is
second makes it work on GTK

Reply:
All changes go to second commit
@aarani

Reply:
Extracted to a separate commit.